x-pack/examples/ui_actions_enhanced_examples/public/drilldowns/dashboard_to_url_drilldown/index.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiCallOut, EuiFieldText, EuiFormRow, EuiSpacer, EuiSwitch } from '@elastic/eui';
+import { reactToUiComponent } from '../../../../../../src/plugins/kibana_react/public';
+import { UiActionsEnhancedDrilldownDefinition as Drilldown } from '../../../../../plugins/ui_actions_enhanced/public';
+import { ChartActionContext } from '../../../../../../src/plugins/embeddable/public';
+import { CollectConfigProps as CollectConfigPropsBase } from '../../../../../../src/plugins/kibana_utils/public';
+import {
+  SELECT_RANGE_TRIGGER,
+  VALUE_CLICK_TRIGGER,
+} from '../../../../../../src/plugins/ui_actions/public';
+import { ActionExecutionContext } from '../../../../../../src/plugins/ui_actions/public';
+import { SerializableState } from '../../../../../../src/plugins/kibana_utils/common';
+
+function isValidUrl(url: string) {
+  try {
+    new URL(url);
+    return true;
+  } catch {
+    return false;
+  }
+}
+
+export type ActionContext = ChartActionContext;
+
+export interface Config extends SerializableState {
+  url: string;
+  openInNewTab: boolean;
+}
+
+type UrlTrigger = typeof VALUE_CLICK_TRIGGER | typeof SELECT_RANGE_TRIGGER;
+
+export type CollectConfigProps = CollectConfigPropsBase<Config, { triggers: UrlTrigger[] }>;
+
+const SAMPLE_DASHBOARD_TO_URL_DRILLDOWN = 'SAMPLE_DASHBOARD_TO_URL_DRILLDOWN';
+
+export class DashboardToUrlDrilldown implements Drilldown<Config, UrlTrigger> {
```

Comment:
This Drilldown is not used? I deleted it with URL drilldown pr. I think we don't need to carry this around

Reply:
Nope, not used, deleted.